components/tools/OmeroPy/src/omero/plugins/sessions.py

```diff
@@ -264,6 +275,73 @@ def _configure_dir(self, parser):
     def help(self, args):
         self.ctx.out(LONGHELP % {"prog": args.prog})
 
+    def open(self, args):
```

Comment:
I think the find_user and find_group methods do this work for you:

https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroPy/src/omero/cli.py#L2445

Reply:
Thanks. I thought there should already be a method like this somewhere. One of the reasons why I don't like Python (or Javascript), because you usually have to dig through files with thousands of lines of code :-/